rust/gui-client/README.md

```diff
@@ -49,6 +49,27 @@ stat ../target/release/Firezone.exe
 stat ../target/release/bundle/msi/Firezone_0.0.0_x64_en-US.msi
 ```
 
+## Signing the Windows MSI in GitHub CI
+
+The MSI is signed in GitHub CI using the `firezone/firezone` repository's
+secrets. This was originally set up using these guides for inspiration:
+
+- https://melatonin.dev/blog/how-to-code-sign-windows-installers-with-an-ev-cert-on-github-actions/
+- https://support.globalsign.com/code-signing/code-signing-using-azure-key-vault
+
+Renewing / issuing a new code signing certificate and associated Azure entities is outside the scope of this section. Use the guides above if this needs to be done.
+
+Instead, you'll most likely simply need to rotate the Azure `CodeSigning` Application's client secret.
+
+To do so, login to the Azure portal using your `@firezoneprod.onmicrosoft.com` account.
+
+Then,
+
+- Find and navigate to the `CodeSigning` app registration.
```

Comment:
I cannot find this. I tried "Trusted Signing Accounts"

<img width="436" alt="image" src="https://github.com/user-attachments/assets/f2c32b18-f8ed-4adb-b48a-d1df936f30ca">


Almost everything I click in the Azure Portal, I get this thing about free trials. Am I in the right place?
<img width="1175" alt="image" src="https://github.com/user-attachments/assets/3c90aff3-9a13-4ced-8936-bf0f1e91be92">

Reply:
Huh, that's not right. Let me double check. Thanks for actually trying these steps!

Reply:
Can you give it a shot now? I think I need to add you to the necessary Roles. Still figuring out what those are.

Reply:
Now I can get into `Trusted Signing Accounts` (Whose URL includes `Microsoft.CodeSigning` so I'm not sure what it's really called lol), but I can't see any.
<img width="898" alt="image" src="https://github.com/user-attachments/assets/bda58c0e-26a3-4d64-baa5-d4658cec6193">

Nothing in `App Services` either
<img width="1100" alt="image" src="https://github.com/user-attachments/assets/b5a2ea4a-a39c-4009-8ff8-d793aba19dbe">

Reply:
Ah, it's in the Entra ID service. Go to Entra ID and then App Registrations on the left.